### ISSUE TITLE
Какую версию мода использовать для fabric 1.19.1

### ISSUE THREAD

**Issue by Klatch13** [KenRouKoro/C3H6N6O6#14]
Beta9 или beta8. Как хорошо он будет работать на 1.19.1?

**Comment by Klatch13** [comment]
@KenRouKoro

**Comment by Klatch13** [comment]
[crash-2022-09-28_08.59.45-server.txt](https://github.com/KenRouKoro/C3H6N6O6/files/9663303/crash-2022-09-28_08.59.45-server.txt)
[latest.log](https://github.com/KenRouKoro/C3H6N6O6/files/9663304/latest.log)

Я использовал beta9

![image](https://user-images.githubusercontent.com/108920477/192738376-908d8451-bb00-4436-afd4-6eacd3e73a9f.png)
список модов

**Comment by Klatch13** [comment]
![2022-09-28_14 22 50](https://user-images.githubusercontent.com/108920477/192742224-9d690e78-c43a-4e80-9971-d511f7c263a3.png)
Я не думаю что он вообще работает на 1.19.1
Я сейчас запустил тестовый сервер только с этим модом. И как видите при 1225 сущностей 11 TPS, хотя серверу выделено 16 ядер с частотой 2.8 если не ошибаюсь

![image](https://user-images.githubusercontent.com/108920477/192742944-c865c506-6fbb-4c9a-87bf-a799aa731fce.png)
[latest.log](https://github.com/KenRouKoro/C3H6N6O6/files/9663476/latest.log)

**Comment by Klatch13** [comment]
[latest.log](https://github.com/KenRouKoro/C3H6N6O6/files/9663473/latest.log)

**Comment by KenRouKoro** [comment]
Не очень хорошо, потому что много производительности расходуется на блокировку.

**Comment by KenRouKoro** [comment]
И с чисто оригинальными расчетами столкновений, я думаю, вы поняли идею.

**Comment by KenRouKoro** [comment]
Кроме того, недавно был разработан план по обеспечению совместимости с литием. Первые испытания показали, что производительность улучшена более чем в восемь раз по сравнению с оригиналом в большом количестве биологических ситуаций. Однако впереди еще много работы, и я всего лишь один человек.

**Comment by Klatch13** [comment]
Я тоже думал, что коробка слишком маленькая и от этого много расчетных столкновений. Про литий, можете дать конфигурацию, я хочу попробовать это)

**Comment by KenRouKoro** [comment]
[libs.zip](https://github.com/KenRouKoro/C3H6N6O6/files/9670360/libs.zip)
![image](https://user-images.githubusercontent.com/21196863/192925593-2fd75d3f-750e-42c2-834c-4f955b38ee7a.png)
Никогда, никогда, никогда, никогда, запускать только в оригинальной версии. И есть огромные ошибки и проблемы с командной системой.

**Comment by KenRouKoro** [comment]
Не используйте команду /kill

**Comment by Klatch13** [comment]
Я сейчас проверю

**Comment by Klatch13** [comment]
А mixin.entites?

**Comment by KenRouKoro** [comment]
Нет необходимости, основная цель новой версии - совместимость с контентом под этим миксином.

**Comment by KenRouKoro** [comment]
![image](https://user-images.githubusercontent.com/21196863/192927525-4f8fc184-ea7d-4273-9cf2-9412ebd22e8e.png)

**Comment by Klatch13** [comment]
У меня запускается вместе с литием

**Comment by Klatch13** [comment]
Запустилось. Странно

**Comment by Klatch13** [comment]
Похоже сучился краш

**Comment by Klatch13** [comment]
[latest.log](https://github.com/KenRouKoro/C3H6N6O6/files/9670477/latest.log)
[crash-2022-09-29_03.00.20-server.txt](https://github.com/KenRouKoro/C3H6N6O6/files/9670478/crash-2022-09-29_03.00.20-server.txt)

**Comment by Klatch13** [comment]
![image](https://user-images.githubusercontent.com/108920477/192928956-95e998d2-c0c7-4bcf-8111-dcf4ae4690d2.png)

**Comment by KenRouKoro** [comment]
![image](https://user-images.githubusercontent.com/21196863/192929226-c654b6b4-5a2e-450c-a0ae-735cd5d4cdc2.png)
Пожалуйста, используйте эту внутреннюю тестовую версию.

**Comment by Klatch13** [comment]
Я не понял

**Comment by KenRouKoro** [comment]
lib.zip содержит версию beta 10.

**Comment by KenRouKoro** [comment]
![image](https://user-images.githubusercontent.com/21196863/192929585-61579f41-20b1-437a-853b-dd14acc34d56.png)

**Comment by Klatch13** [comment]
А где его скачать?

**Comment by KenRouKoro** [comment]
![image](https://user-images.githubusercontent.com/21196863/192929615-b6e2cc0b-8538-4282-8dc9-13d242d307a9.png)
Здесь, поскольку github не позволяет загружать файлы в формате jar в выпуски

**Comment by Klatch13** [comment]
Скачал, я просто не заметил это)

**Comment by Klatch13** [comment]
Вроде все отлично работает, но у меня появляется пинг

**Comment by Klatch13** [comment]
![2022-09-29_08 31 00](https://user-images.githubusercontent.com/108920477/192932346-cfee7649-7234-46ed-b2d7-722dc9e09aa8.png)

**Comment by Klatch13** [comment]
![image](https://user-images.githubusercontent.com/108920477/192932401-58cf406e-d445-489f-b221-332b9a604bf4.png)

**Comment by Klatch13** [comment]
2000 криперов

**Comment by Klatch13** [comment]
пинг сильно увеличился, но у 2 моих друзей он не увеличивался

**Comment by KenRouKoro** [comment]
Какой тип процессора вы используете, просто спрашиваю.

**Comment by Klatch13** [comment]
Я использую сервер от OracleCloud. И если не ошибаюсь, то там стоит процессор ampere altra 80c

**Comment by KenRouKoro** [comment]
ARM SERVER ? Производительность одноядерных вычислений очень низкая.

**Comment by Klatch13** [comment]
Мне доступно только 16 CPUs

**Comment by Klatch13** [comment]
> ARM SERVER ? Производительность одноядерных вычислений очень низкая.

Я особо в этом не разбираюсь)

**Comment by Klatch13** [comment]
Но в любом случае за бесплатно такой сервер это хорошо

**Comment by Klatch13** [comment]
> ARM SERVER ? Производительность одноядерных вычислений очень низкая.

Я читал про процессор и там что то было сказано про ARM

**Comment by KenRouKoro** [comment]
ampere altra 80c - это процессор архитектуры arm (она же lite instruction set) с очень низкой одноядерной производительностью, но архитектура arm позволяет легко создавать многоядерные процессоры с низким энергопотреблением.
Он не очень подходит для серверов Minecraft.

**Comment by Klatch13** [comment]
Да, если не ошибаюсь, то серверу майнкрафт нужна большая одноядерная производительность, но в целом мне этого хватает. Я думаю на моем сервере будет играть человек 20

**Comment by wicivo** [comment]
![image](https://user-images.githubusercontent.com/116191734/196772091-631512f8-4c3c-4f2b-96f3-a2c6e0a106c3.png)

**Comment by wicivo** [comment]
Я пишу со второго аккаунта

**Comment by wicivo** [comment]
Мод работает
Но как только я сделал этот скрин, появилось это 
![image](https://user-images.githubusercontent.com/116191734/196772423-3ddb84bc-a515-43c6-9035-061ca36555d0.png)

**Comment by wicivo** [comment]
Кстати ваш мод не совместим с VMP. Возможно ли сделать так, чтобы они были совместимы?

**Comment by wicivo** [comment]
![image](https://user-images.githubusercontent.com/116191734/196777035-ae3f63fe-d0ab-43c9-b6a1-6767b89e0ee8.png)
Это я тестировал на сервере от OracleCloud, но только с 4 ядрами, а не с 16. Также есть другие моды на оптимизацию. Но видно, что мод работает, иначе бы процессор не нагрузился на 70 процентов и TPS был бы маленьким. где-то 1000+ овец

**Comment by wicivo** [comment]
Советую ваш мод к использованию. Вы хорошо постарались) Спасибо

**Comment by wicivo** [comment]
> Мод работает Но как только я сделал этот скрин, появилось это ![image](https://user-images.githubusercontent.com/116191734/196772423-3ddb84bc-a515-43c6-9035-061ca36555d0.png)
Думаю на это не стоит обращать внимание, т.к на сервере OracleCloud этого не было
